src/containers/sign-on/store/sagas.js

```diff
@@ -432,6 +457,12 @@ function* followArtists() {
       const userIndex = userIdsToFollow.findIndex(fId => fId === userId)
       if (userIndex > -1) hasFollowConfirmed[userIndex] = true
     }
+
+    // Auto-follow Hot & New Playlist
+    if (IS_PRODUCTION) {
```

Comment:
almost there, but we should do this in parallel w/ the previous follow users. that would be a 

yield fork(...) above when we do that work. or alternatively pulling that work into a forked saga